evdev/eventio_async.py

```diff
@@ -85,8 +85,7 @@ def __next__(self):
     def __aiter__(self):
         return self
 
-    @asyncio.coroutine
-    def __anext__(self):
+    async def __anext__(self):
```

Comment:
according to the python docs [`__anext__`](https://docs.python.org/3/reference/datamodel.html#object.__anext__) should return a Awaitable which this function already does. So there is no need to use the async keyword. `async def` would wrap the whole function again inside a awaitable making it necessary to await this function twice.
```suggestion
    def __anext__(self):
```

As for the `@asyncio.coroutine`: I think this was never needed as it was meant for [generator-based coroutines](https://docs.python.org/3.10/library/asyncio-task.html?highlight=asyncio%20coroutine#generator-based-coroutines) which this function is not.

I did some quick tests with pythin3.8 and 3.10. This seems to work fine.

Reply:
Hi. I think this is the correct solution indeed. Will release a new version asap. Thanks!